Support/Multitenancy/Multiple-Ironic-conductors/bmo-config/kustomization.yaml

```
@@ -0,0 +1,36 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: baremetal-operator-system
+resources:
+- https://github.com/metal3-io/baremetal-operator/config/overlays/basic-auth_tls?ref=release-0.6
+
+configMapGenerator:
+- name: ironic
+  behavior: create
+  envs:
+  - ironic.env
+
+patches:
+- patch: |
+    # Don't try to pull again the pre-loaded image
+    - op: replace
+      path: /spec/template/spec/containers/0/imagePullPolicy
+      value: IfNotPresent
+  target:
+    kind: Deployment
+    name: controller-manager
+
+images:
+- name: quay.io/metal3-io/baremetal-operator
+  newTag: release-0.5
+
+# We cannot use suffix hashes since the kustomizations we build on
+# cannot be aware of what suffixes we add.
+generatorOptions:
+  disableNameSuffixHash: true
+
+secretGenerator:
+  - name: ironic-credentials
```

YAML formatting, please indent lists the k8s way.

```suggestion
- name: ironic-credentials
```